index.html

```diff
@@ -197,6 +197,7 @@
             <h4>[[title]]</h4>
             <p class="repoLink"><a href="[[repoURL]]" target="_blank">[[repoTitle]]</a></p>
             <p>[[description]]</p>
+            <p><b>Topics:</b><br/>[[topics]]</p>
```

Comment:
add `[[#if topics]] ... [[/if]]` so that this section is hidden when no topics are collected

Reply:
I worked that in.
A bit unsure if I get that IF-replacement logic right but it seems to work.
See: https://github.com/SAP/project-portal-for-innersource/pull/12/files#diff-7a094af86da034414240a174bdfb685fe3cf7c50ececa1b23d4b3e91523554edR286-R288